src/lib/layer-manager.js

```diff
@@ -115,9 +115,11 @@ export default class LayerManager {
     assert(this.context.viewport, 'LayerManager.drawLayers: viewport not set');
 
     const {uniforms} = this.context;
+    let drawCount = 0;
```

Comment:
Good point. Ideally it should be the accumulated number of draw calls. Which could be more than 1 for some layers. But that would require layers to return draw call counts, or let model.draw to have this support built in (maybe not a bad idea).